### PR TITLE
New package: sshguard-1.6.3

### PR DESCRIPTION
**Pull request by lodvaer**

### CODE REVIEW

**Comment by lodvaer** [comment]
Upstream issue causes the build failure, it seems fixed in HEAD. 1.6.3 should be out soon. (I'll also remove the mistaken makedepends-value then.)

**Comment by Vaelatern** [comment]
[1.6.3 is out](https://bitbucket.org/sshguard/sshguard/commits/tag/v1.6.3)

**Comment by lodvaer** [comment]
It has to manipulate iptables, so I think getting it to run as a user other than root is somewhat complicated.

sshguard must be configured to read logs somehow, and the runit-file was intended to be included for convenience if used with socklog. Should it perhaps be renamed to sshguard-socklog (like with dhcpcd-eth0)? And if so, should `sv check socklog-unix || exit 1` be kept?

File: srcpkgs/sshguard/files/sshguard/run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sv check iptables >/dev/null || exit 1
+exec sshguard -l /var/log/socklog/secure/current -b 200:/var/db/sshguard/blacklist.db 2>&1
```
Comment (Vaelatern) [inline srcpkgs/sshguard/files/sshguard/run]: Why are you writing to socklog's file? Log to your own file, or the syslog if that's an option.

Comment (chneukirchen) [inline srcpkgs/sshguard/files/sshguard/run]: ... that's the input file.

Comment (Vaelatern) [inline srcpkgs/sshguard/files/sshguard/run]: I guess in which case it makes sense to check for socklog.

**Comment by Vaelatern** [comment]
On furthur review, yes, your runit script depends on socklog being present and running, so that `sv` check makes sense, as well as renaming your sample runit script to `sshguard-socklog`

File: srcpkgs/sshguard/patches/patch-src_sshguard_logsuck.c [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+Logging that one reads from the log causes an infinite loop.
+
+--- src/sshguard_logsuck.c.orig	2015-11-11 23:00:05.096089572 +0000
++++ src/sshguard_logsuck.c	2016-03-09 19:18:06.326044898 +0000
+@@ -286,7 +286,7 @@
```
Comment (Vaelatern) [inline srcpkgs/sshguard/patches/patch-src_sshguard_logsuck.c]: `+@@ -286,7 +286,6 @@`

File: srcpkgs/sshguard/patches/patch-src_sshguard_logsuck.c [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+     }
+     list_iterator_stop(& sources_list);
+ 
+-    sshguard_log(LOG_INFO, "Refreshing sources showed %u changes.", numchanged);
++    /* sshguard_log(LOG_INFO, "Refreshing sources showed %u changes.", numchanged); */
```
Comment (Vaelatern) [inline srcpkgs/sshguard/patches/patch-src_sshguard_logsuck.c]: Remove this line, not necessary.

**Comment by lodvaer** [comment]
Ok. runit file updated and renamed. I also included a patch I found necessary (upstream issue number 25).

**Pull request merged**